pkg/podexecutor/staticpod.go

```diff
@@ -64,7 +64,13 @@ func (s *StaticPod) APIServer(ctx context.Context, etcdReady <-chan struct{}, ar
 	if err := images.Pull(s.PullImages, "kube-apiserver", s.Images.KubeAPIServer); err != nil {
 		return nil, nil, err
 	}
-
+	args = append(args,
+		`--basic-auth-file=""`,
```

Comment:
`basic-auth-file` needs to be explicitly set to empty? if so, please comment as to why

Reply:
Updated. Thanks!